### ISSUE TITLE
I cannot set size for SVG output..

### ISSUE THREAD

**Issue by GYiyouth** [nidi3/graphviz-java#129]
cannot set width or height for svg
```java
            Graphviz.useEngine(new GraphvizCmdLineEngine(), new GraphvizV8Engine()); // Rasterizer.builtIn() works only with CmdLineEngine
            Graph g = graph("example5").directed()
                    .with(node("abc").link(node("xyz")))
                    ;

            Graphviz viz = Graphviz.fromGraph(g);
            viz.height(180).render(Format.SVG).toFile(new File("example/ex5.svg"));
            viz.width(200).rasterize(Rasterizer.BATIK).toFile(new File("example/ex5b.png"));
            viz.width(200).rasterize(Rasterizer.SALAMANDER).toFile(new File("example/ex5s.png"));
            viz.width(200).rasterize(Rasterizer.builtIn("png")).toFile(new File("example/ex5p"));
            String dot = viz.render(Format.DOT).toString();
            String json = viz.engine(Engine.NEATO).render(Format.JSON).toString();
            BufferedImage image = viz.render(Format.PNG).toImage();
```

it will error
![image](https://user-images.githubusercontent.com/12520517/67301057-7b210c00-f521-11e9-916c-e68a02a147b9.png)

and size of svg not changed

![image](https://user-images.githubusercontent.com/12520517/67301148-99870780-f521-11e9-9d5b-e2c7cb8b731d.png)

**Comment by GYiyouth** [comment]
My Java version is 

java version "1.8.0_191"
Java(TM) SE Runtime Environment (build 1.8.0_191-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.191-b12, mixed mode)

I run program on Mac

**Comment by nidi3** [comment]
The first error is because you don't have native graphviz installed on your computer. In this case, `GraphvizV8Engine` is used as fallback which does not work together with `Rasterizer.builtIn()`.
The unchanged size is indeed a bug.

**Comment by nidi3** [comment]
Should be fixed in version `0.11.1`.
